_build/go.mod

```diff
@@ -4,9 +4,10 @@ module github.com/os-observability/redhat-opentelemetry-collector
 
 go 1.21.0
 
-toolchain go1.21.8
+toolchain go1.21.1
```

Comment:
why are we downgrading?

Reply:
I think we should bump go to 1.22 which is used by productization

Reply:
It's what I have installed on my machine. I didn't notice about this. Change. I'll revert.